packages/cli/lib/lib/webpack/transform-config.js

```diff
@@ -15,11 +15,6 @@ module.exports = async function (env, config, ssr=false) {
 		throw new Error(`preact-cli config could not be loaded!\nFile ${env.config} not found.`);
 	}
 
-	require('@babel/register')({
-		presets: [[require.resolve('@babel/preset-env'), {
-			"targets": { "node": "current" }
-		}]]
-	});
```

Comment:
This will break `preact.config.js` pretty much everywhere, are we gonna change something about that as well?

Reply:
Oh, derp. I've always done `module.exports` and forgot we allowed that. If it were up to me, we'd drop it entirely. 

Will add it back.